config/locales/defaults/ca.yml

```diff
@@ -15,7 +15,7 @@ ca:
     - dv
     - ds
     abbr_month_names:
-    -
+    - 
```

Comment:
@ferblape is it possible that this always happen due to the missing tabulator char? Perhaps, should it be??
```yml
abbr_month_names:
    - gen
    - feb
```
Just asking

Reply:
mmm, I think that guilty is my Atom default configuration because. Atom is trailing all withespaces every time that save the file. 

1. I will disable (auto delete end whitespaces)
https://flight-manual.atom.io/using-atom/sections/editing-and-deleting-text/#whitespace
2. I will enable highlights spaces like this
https://atom.io/packages/trailing-spaces

Reply:
No, no, we know the `check_i18n` script is the reason to set/unset the tabulation. But it's a recurrent issue that I cannot figure out how it happens, sometimes there's empty row, sometimes there isn't :(

Reply:
it's not an empty row but an empty space in the row,  I don't know what causes it, probably an editor linter?